go-controller/pkg/ovn/external_ids_syncer/acl/acl_sync_test.go

```diff
@@ -69,6 +69,11 @@ func joinACLName(substrings ...string) string {
 	return strings.Join(substrings, "_")
 }
 
+func buildPortGroup(hashName, name string, ports []*nbdb.LogicalSwitchPort, acls []*nbdb.ACL) *nbdb.PortGroup {
```

Comment:
the sync should never be updated, because the code only works with the previous version of ids, so I think you can just revert changes of this file and it should work

Reply:
I made this change because it does not compile